client.go

```diff
@@ -21,6 +24,8 @@ type Client struct {
 	MsgCount   int
 	MsgQoS     byte
 	Quiet      bool
+	nTopics    int
```

Comment:
All other fields are exported, so let's export this one too and name it `NumTopics`

Reply:
Fixed all the references of `nTopics` to `NumTopics`